src/lib.rs

```diff
@@ -172,81 +172,63 @@
 //! [`Running`]: service::ServiceState::Running
 
 #![cfg(windows)]
-// Because of how deeply error-chain recurse with this many error types.
-#![recursion_limit = "128"]
 
 #[macro_use]
-extern crate error_chain;
+extern crate err_derive;
+pub type Result<T> = std::result::Result<T, ErrorKind>;
 
+#[derive(Error, Debug)]
+pub enum ErrorKind {
```

Comment:
Please rename to `Error` now when it's an error (with causes and everything), and not just a kind.

Reply:
forgot about that in the process of refactoring, thanks